ads/_config.js

```diff
@@ -333,6 +333,8 @@ const adConfig = jsonConfiguration({
     prefetch: 'https://cdn.broadstreetads.com/init-2.min.js',
   },
 
+  'byplay': {},
```

Comment:
I see you are not using `renderStart` here. If you implement it in your external js you should see some gains in render speed, but ultimately up to you.

Reply:
I removed this code, but it didn't work. 🤔

Reply:
I was referring to `render-start` as outlined here: https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis

It is optional, and can always be done in a follow up PR.